desktop/core/src/desktop/settings.py

```diff
@@ -117,6 +118,8 @@
 # Examples: "http://media.lawrence.com/media/", "http://example.com/media/"
 MEDIA_URL = ''
 
+if ENABLE_DIRECT_UPLOAD:
```

Comment:
Shouldn't it be `ENABLE_DIRECT_UPLOAD.get()`?

Reply:
This doesn't look like a `generic configuration`, I think we should make a separate section for Indexer like at the end for other specific configs and put it there.

Reply:
1. Thanks @Harshg999.

2. This variable comes under the core settings that's why I putted that variable in  generic configuration. any thoughts @ranade1 @wing2fly ?